about.html

```diff
@@ -1,73 +1,60 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
-  <meta charset="utf-8">
+    <meta charset="utf-8">
 
-  <title>Sprint Challenge - About</title>
+    <title>Sprint Challenge - About</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
+<section>
+    <img src="https://raw.githubusercontent.com/LambdaSchool/web-Sprint-Challenge-User-Interface-and-Git/master/img/lambda-black.png">
```

Comment:
Nice work on the nav area!  Don't forget to change your home link back to index.html.  Also, You created a section for this area, there are no rules about not doing so, but I will link a article that shows the ways semantic HTML should theoretically get laid out.  I would make the section a header or just use a nav.